app/router.js

```diff
@@ -7,6 +7,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('reminders'), { path: '/reminders'};
```

Comment:
If your route is `reminders` you don't need the path to also specify reminders - it will do this by default. This means you can remove the `{ path: '/reminders'}` bit.